### PR TITLE
add standardize 400 and 422 error

### PR DESCRIPTION
**Pull request by hanjuTsai**

### CODE REVIEW

**Review by alixodendhal — COMMENTED** [review]
Are we missing a field? 
I remember there is a `field` parameter in our unified error structure to call out which field may have an error. 

[unified-errors-spec.yaml](https://bitbucket.org/instadevelopers/unified-errors/src/618cbb26c821fa5cf7d0ecb0f594e2a6397ff1b2/src/main/resources/unified-errors-spec.yaml#lines-16)

**Comment by hanjuTsai** [comment]
yes. I will add the field.

**Review by alixodendhal — APPROVED** [review]

File: version/1.0.0/unified-errors-1.0.0.yaml [hunk 1]
```diff
@@ -83,6 +83,39 @@ components:
...
+        code:
+          type: string
+          default: invalid_input
+          enum:
+            - invalid_input
+        description:
+          type: string
+          default: "Syntatically incorrect input provided"
+          enum:
+            - "Syntatically incorrect input provided"
```
Comment (basonjar) [inline version/1.0.0/unified-errors-1.0.0.yaml]: Typo in default value and enum value "Syntactically"

**Comment by alixodendhal** [comment]
can we merge now?

**Review by basonjar — APPROVED** [review]

**Pull request merged**